client/src/modules/stock/lots/registry.service.js

```diff
@@ -46,6 +46,11 @@ function LotsRegistryService(uiGridConstants, Session) {
       displayName : 'TABLE.COLUMNS.INVENTORY_GROUP',
       headerTooltip : 'TABLE.COLUMNS.INVENTORY_GROUP',
       headerCellFilter : 'translate',
+    }, {
+      field : 'barcode',
+      displayName : 'LOTS.BARCODE',
+      headerTooltip : 'LOTS.BARCODE',
+      headerCellFilter : 'translate',
```

Comment:
Can you put this in a cell template with a `bh-reference-link` that opens the barcode modal automatically when we click it?

Reply:
Also, let's set `visible: false` by default.